lib/diaspora_federation/validators/relayable_validator.rb

```diff
@@ -0,0 +1,11 @@
+module DiasporaFederation
+  module Validators
+    module RelayableValidator
+      def self.included(model)
+        model.class_eval do
+          rule :parent_guid, :guid
```

Comment:
Actually my tests show an opposite thing: parent_author_signature is mandatory and author_signature is optional. Here is logs:
[Both parameters are empty](https://gist.github.com/cmrd-senya/874abcaf75254e39d7cc#file-both-empty-log-L14).
[Both parameters are filled](https://gist.github.com/cmrd-senya/3212e954eec9801424da#file-both-filled-log-L10).
[Only author_signature is empty](https://gist.github.com/cmrd-senya/cd5669b86b598cd9554b#file-author_signature-empty-log-L10).
[Only parent_author_signature is empty](https://gist.github.com/cmrd-senya/d9b7bcad9c583bed1888#file-parent_author_signature-empty-log-L10).

Maybe that means a bug in Diaspora?